akka-docs/rst/scala/code/docs/akka/typed/IntroSpec.scala

```diff
@@ -66,6 +66,28 @@ object IntroSpec {
         }
       }
     //#chatroom-behavior
+
+    //#chatroom-actor
+    def mutableChatRoom(): Behavior[Command] =
+      Deferred[Command] { _ =>
+        var sessions: List[ActorRef[SessionEvent]] = List.empty
+        Stateful[Command] { (ctx, msg) ⇒
```

Comment:
Hmm, this should actually be Stateless, but of course I see the irony in that. (same applies to Java version above)